### ISSUE TITLE
Bottom Tabs Content Animation

### ISSUE THREAD

**Issue by tejas77** [wix/react-native-navigation#7396]
## ❓ Questions and Help
By using bottomTabs and animation, How can I achieve this type of animation ?

![Tab Animation](https://i.stack.imgur.com/obGJI.gif)

### Please note that this issue tracker is not a help form and this issue will be closed.

Please contact us instead. We have a few channels:

- [Stack Overflow](https://stackoverflow.com/questions/tagged/wix-react-native-navigation)
- [Discord](https://discord.gg/DhkZjq2)

**Comment by tejas77** [comment]
No one has given answer for this question in stackoverflow for years, that's why asking over here.

**Comment by sabin10** [comment]
I don't know a way to achieve the 'pager transition' using RNN. But I was able to achieve this by removing bottom tabs RNN implementation from the root screen and using [react-native-tab-view](https://github.com/satya164/react-native-tab-view) + creating a custom tab-bar which I placed at the bottom.

But this comes with cons, the biggest offender being that when pushing another screen over the root, the bottom tabs are hidden underneath the new pushed screen.

For pros, now you will have a custom bottom-tab-bar because it's implemented on react-native side, instead of using images as tab icons.
